MAINTAINERS.yml

```diff
@@ -0,0 +1,1008 @@
+    # Provisional entry, to be corrected
+    status: maintained
+    collaborators:
+        - trond-snekvik
+    files:
+        - subsys/bluetooth/mesh/
+    labels:
+        - "area: Bluetooth Mesh"
+
+"Build system":
+    status: maintained
+    maintainers:
+        - SebastianBoe
+    collaborators:
+        - nashif
+    files:
+        - cmake/
+    files-regex:
+        - CMakeLists.txt$
+        - \.cmake$
+    labels:
+        - "area: Build System"
+
+"C library":
+    # Provisional entry, to be corrected
+    status: orphaned
+    collaborators:
+         - nashif
+         - andrewboie
+    files:
+         - lib/libc/
+    labels:
+         - "area: C Library"
+
+"CMSIS API layer":
+    # Provisional entry, to be corrected
+    status: orphaned
+    collaborators:
+         - nashif
+    files:
+         - lib/cmsis_rtos_v*/
+
+Debug:
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - nashif
+    files:
+        - include/debug/
+        - subsys/debug/
+        - tests/subsys/debug/
+    labels:
+        - "area: Debugging"
+
+DFU:
+    # Provisional entry, to be corrected
+    # TODO: spreadsheet not matching CODEOWNERS
+    status: maintained
+    maintainers:
+        - nvlsianpu
+    files:
+        - include/dfu/
+        - subsys/dfu/
+        - tests/subsys/dfu/
+
+Devicetree:
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - galak
+    collaborators:
+        - ulfalizer
+    files:
+        - scripts/dts/
+    labels:
+        - "area: Device Tree"
+
+"Display drivers":
+    # Provisional entry, to be corrected
+    status: orphaned
+    collaborators:
+        - vanwinkeljan
+    files:
+        - drivers/display/
+        - dts/bindings/display/
+        - include/display.h
+        - include/display/
+        - include/drivers/display.h
+    labels:
+        - "area: Display"
+
+Documentation:
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - dbkinder
+    files:
+        - doc/
+    files-regex:
+        - \.rst$
+        - /doc/
+    labels:
+        - "area: Documentation"
+
+"Drivers: ADC":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - anangl
+    files:
+        - drivers/adc/
+        - include/drivers/adc.h
+    labels:
+        - "area: ADC"
+
+"Drivers: ADC stm32":
+    # Provisional entry, to be corrected
+    collaborators:
+        - cybertale
+    files:
+        - drivers/adc/adc_stm32.c
+        - dts/bindings/iio/adc/st*stm32-adc.yaml
+
+"Drivers: Audio":
+    # Provisional entry, to be corrected
+    # TODO: spreadsheet not matching CODEOWNERS
+    status: maintained
+    maintainers:
+        - Vudentz
+    files:
+        - drivers/audio/
+        - include/audio/
+
+"Drivers: CAN":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - alexanderwachter
+    files:
```

Comment:
collaborators:
    - karstenkoenig
    - henrikbrixandersen

Reply:
Thanks @alexanderwatcher :). Just a procedural check: @karstenkoenig @henrikbrixandersen : Are you ok with being added here?

Reply:
Absolutely. Thank you.

Reply:
Totally, thanks!